tests/mapper/test_node_mappers.py

```diff
@@ -112,6 +112,44 @@ def example_hin_3(feature_size_by_type=None):
     return G, nodes_type_1, nodes_type_2
 
 
+def assert_sequence_shuffle(seq, shuffle, max_iter=5):
+    """
+    Assert whether the sequence object shuffles as expected when calling ``on_epoch_end``
+
+    Args:
+        seq (tensorflow.keras.Sequence): Sequence object
+        shuffle (bool): True if shuffle is turned on
+        max_iter(int): Number of times to check shuffling behaviour.
+    """
+
+    def flatten_features(seq):
+        # check (features == labels) and return flattened features
+        batches = [
+            (np.ravel(seq[i][0][0]), np.array(seq[i][1])) for i in range(len(seq))
+        ]
+        features, labels = zip(*batches)
+        features, labels = np.concatenate(features), np.concatenate(labels)
+        assert all(features == labels)
```

Comment:
I think this can be:

```suggestion
        assert np.array_equal(features, labels)
```